balancer/mist/mist_balancer.go

```diff
@@ -339,8 +340,11 @@ func fallbackNode(fallbackAddr, fallbackPrefix, playbackID, defaultPrefix string
 }
 
 // make a balancing request to MistUtilLoad, returns a server suitable for playback
-func (b *MistBalancer) MistUtilLoadBalance(ctx context.Context, stream, lat, lon string) (string, error) {
-	str, err := b.mistUtilLoadRequest(ctx, "/", stream, lat, lon)
+func (b *MistBalancer) MistUtilLoadBalance(ctx context.Context, stream, lat, lon string, urlSuffix ...string) (string, error) {
+	// add `tag_adjust={"region_name":1000}` to bump current region weight so it's more important that other params like
+	// cpu, memory, bandwidth or distance. It's meant to minimise redirects to other regions after current region is "selected"
+	// by the DNS rules.
+	str, err := b.mistUtilLoadRequest(ctx, "/", stream, lat, lon, fmt.Sprintf("&tag_adjust={\"%s\":1000}", b.config.OwnRegion))
```

Comment:
Should this be `?tag_adjust` instead of `&tag_adjust`? Unless there are other query params being passed in the request but doesn't look like it?

Reply:
Right, right. Fixed and tested.